internal/resource/obtenant/utils.go

```diff
@@ -920,3 +922,91 @@ func CreateUserWithCredentials(m *OBTenantManager) error {
 	}
 	return nil
 }
+
+func (m *OBTenantManager) listOBParameters() (*v1alpha1.OBParameterList, error) {
+	obparameterList := &v1alpha1.OBParameterList{}
+	err := m.Client.List(m.Ctx, obparameterList, client.MatchingLabels{
+		oceanbaseconst.LabelRefOBCluster: m.OBTenant.Spec.ClusterName,
```

Comment:
Why not list the parameters with UID field of OBTenant as `RefUID` label?

Reply:
These labels both work, I just follow the logic of obcluster manager.

Reply:
I think it's a good idea to use another label, and do not set the cluster name label when it's a tenant parameter

Reply:
> These labels both work, I just follow the logic of obcluster manager.

Listing parameters with only cluster label will get more data than tenant manager need. I recommend to use tenant `uid` as the filtering label or both of they two. :)

Reply:
Yes, that's right, I have changed the label of obparameter which is created with obtenant, use the label ref-obtenant instead of ref-obcluster, thus when the obcluster manager list the parameters, it will only get the relevant parameters, but I still keep the filter logic.